cratedb_toolkit/io/dynamodb/copy.py

```diff
@@ -54,12 +53,16 @@ def start(self):
         """
         Read items from DynamoDB table, convert to SQL INSERT statements, and submit to CrateDB.
         """
-        records_in = self.dynamodb_adapter.count_records(self.dynamodb_table)
+        records_in = self.dynamodb_adapter.count_records(table_name=self.dynamodb_table)
         logger.info(f"Source: DynamoDB table={self.dynamodb_table} count={records_in}")
 
+        # FIXME!
```

Comment:
Do we need a comment here on what is open to be fixed? At least I wouldn't know.